logging/google/cloud/logging/logger.py

```diff
@@ -194,17 +205,23 @@ def log_text(self, text, client=None, labels=None, insert_id=None,
         :param http_request: (optional) info about HTTP request associated with
                              the entry
 
+        :type resource :class:`google.cloud.logging.resource.Resource`
```

Comment:
You are missing a colon after `resource` (here and most of the other places you've added a parameter). Should be:

```
        :type resource: :class:`~google.cloud.logging.resource.Resource`
        :param resource: (Optional) Monitored resource of the entry, defaults
                         to the global resource type.
```

I'm fairly certain this will break the docs build?